### PR TITLE
Search locationbar

### PR DESCRIPTION
**Pull request by maharjunm**

### CODE REVIEW

File: src/components/Location/Location.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.location{
```
Comment (vadlachary2002) [inline src/components/Location/Location.scss:1]: Input box is overflown

File: src/components/SearchBar/SearchBar.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+.searchInput{
+    border: 0.5px solid black;
+    padding :10px;
+    width:100%;
```
Comment (vadlachary2002) [inline src/components/SearchBar/SearchBar.scss]: location input box also overflown

File: src/components/SearchBar/SearchBar.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+}
+.searchResult{
+    background-color: white;
+    padding:10px;
+    width:100%;
+    position:absolute;
+    z-index:1;
+}
+#options:hover{
+    background-color: yellow;
```
Comment (suchitraguduru) [inline src/components/SearchBar/SearchBar.scss]: Change dropdown color from yellow to other light color.

File: src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
-
-export default Home;
+import React from 'react';
+import { ErrorBoundary,Searchbar, Location } from '../../components';
+import data from '../../components/SearchBar/data';
+
+const Home = () => {
+    return (
+        <ErrorBoundary>
+            <Searchbar data={data} />
```
Comment (suchitraguduru) [inline src/pages/Home/Home.tsx]: Set position of  search bar and location as flex.

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "editor.tabSize": 2,
```
Comment (vadlachary2002) [inline .vscode/settings.json]: This file & folder .vscode can be removed

**Comment by Mohammed-Amrath** [comment]
I'LL fix the positions of input boxes and make them a responsive

**Comment by amrathmd** [comment]
Screenshot from 2023-01-27 17-12-41

File: .vscode/settings.json (hunk 5 above)
Comment (maharjunm) [reply .vscode/settings.json]: you can remove this file.

**Review by vadlachary2002 — APPROVED** [review]

**Pull request merged**